src/backend/distributed/transaction/remote_transaction.c

```diff
@@ -418,34 +418,24 @@ void
 FinishRemoteTransactionAbort(MultiConnection *connection)
 {
 	RemoteTransaction *transaction = &connection->remoteTransaction;
-	PGresult *result = NULL;
 	const bool dontRaiseErrors = false;
```

Comment:
small note: we're changing this to `raiseErrors` in https://github.com/citusdata/citus/pull/2123